doctrinal-works-v3/original/38-chapter-38.adoc

```diff
@@ -44,24 +44,40 @@ This is enough to let you understand my will, if you will be so wise as to know
 [.blurb]
 === An answer to the speech or declaration of the great Turk, etc. By G. F.
 
+[.embedded-content-document.letter]
+--
+
+[.salutation]
 Sultan Mahomet,
 
 I find in a paper, which is said to be thy declaration, directed to Leopold,
 the German emperor, dated the 25th of March, 1683, viz.
 
-First, thou sayst thou art a '`possessor of the sepulchre of your God,`' to wit,
+[.numbered-group]
+====
+
+[.numbered]
+_First,_ thou sayst thou art a '`possessor of the sepulchre of your God,`' to wit,
 the christians.
 And that thou art a great persecutor of the christians.
 
-Secondly, thou sayst thou art lord of all the emperors of the world,
+[.numbered]
+_Secondly,_ thou sayst thou art lord of all the emperors of the world,
 from the rising of the sun to the going down thereof.
 
-Thirdly, thou sayst thou art king of all kings, and lord of the tree of life.
+[.numbered]
+_Thirdly,_ thou sayst thou art king of all kings, and lord of the tree of life.
 
-Fourthly, thou sayst thou art commander and guardian of your crucified God, (to wit,
+[.numbered]
+_Fourthly,_ thou sayst thou art commander and guardian of your crucified God, (to wit,
 the christians,) and thou sayst, thou wilt '`pursue your crucified God,
 whose wrath I fear not, nor his coming to defend or deliver you out of my hands.`'
 
+====
+
+[.numbered-group]
+====
+
 Now, here is something in answer to the words of sultan Mahomet,
```

Comment:
It seems like this numbered group starts here, and it's referring to the number group above it in order, but I'm not totally sure if this is where you'd want to start the numbered group.

Reply:
Hmmm... i looked at it for a little while, and I think i slightly prefer moving the start of this group to above the line that says: `But the emperor of the Turks saith` (line 120).  Because the other parts of the numbered-group seem to be enumerating what the emperor _says_, so that feels more like the right spot to begin.  so, why don't you move it down there (and be sure to add a `[.numbered]` to the first paragraph as well. 👍